### DISCUSSION TITLE
Classic mod slider behavior breaks some 2b maps

### DISCUSSION THREAD

**Discussion by PercyDan54** [ppy/osu#27843] [Q&A] [answered]
There's this code in OsuModClassic:
https://github.com/ppy/osu/blob/b7b3401b347e3942956e0292440bb11877d2d294/osu.Game.Rulesets.Osu/Mods/OsuModClassic.cs#L90-L99

Which I found may cause some 2b maps to break. I commented out that method and the replay don't combo break anymore

Example broken maps:
https://osu.ppy.sh/beatmapsets/982020#osu/2055234
![osu_2024-04-13_15-24-53](https://github.com/ppy/osu/assets/50285552/5ffd9dbf-69f1-48f3-8e1c-395fcb660213)
https://osu.ppy.sh/beatmapsets/756794
![osu_2024-04-13_15-25-58](https://github.com/ppy/osu/assets/50285552/74b15ae4-1eca-4b03-9d20-75675e7e65e8)

Pick a Stable replay from the leaderboard and watch
e.g:
https://osu.ppy.sh/scores/2571598160
https://osu.ppy.sh/scores/osu/2655875176

**Comment by Stoppedpuma** [comment, marked as answer]
Duplicate of #27270
